.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        java: [8, 9, 10, 11, 12, 13, 14, 15]
+        java: [8, 9, 10, 11, 12, 13, 14, 15, 16, 17]
```

Comment:
Maybe we should only care for the "Major" Java Versions: `8`, `11`, `16`, `17`. Actions minutes are important and anyone running 9, 10, 13, 14, 15 should just upgrade anyway if they somehow have issues.

Reply:
Good point for this one.